src/libs/actions/Transaction.js

```diff
@@ -1,5 +1,21 @@
+import _ from 'underscore';
 import Onyx from 'react-native-onyx';
+import lodashGet from 'lodash/get';
+import lodashCloneDeep from 'lodash/cloneDeep';
 import ONYXKEYS from '../../ONYXKEYS';
+import * as CollectionUtils from '../CollectionUtils';
+
+const allTransactions = {};
+Onyx.connect({
+    key: ONYXKEYS.COLLECTION.TRANSACTION,
+    callback: (transactions, key) => {
+        if (!key || !transactions) {
+            return;
+        }
+        const transactionID = CollectionUtils.extractCollectionItemID(key);
+        allTransactions[transactionID] = transactions;
```

Comment:
Isn't this one transaction, not multiple? I suggest renaming transactions => transaction